pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -629,13 +651,72 @@ func (b *OFBridge) RetryInterval() time.Duration {
 	return b.retryInterval
 }
 
+func (b *OFBridge) queryTableFeatures() {
+	mpartRequest := &openflow13.MultipartRequest{
+		Header: openflow13.NewOfp13Header(),
+		Type:   openflow13.MultipartType_TableFeatures,
+		Flags:  0,
+	}
+	mpartRequest.Header.Type = openflow13.Type_MultiPartRequest
+	mpartRequest.Header.Length = mpartRequest.Len()
+	tableFeatureChan := make(chan *openflow13.MultipartReply)
+	b.mpartRplChs[mpartRequest.Xid] = tableFeatureChan
+	go func() {
+		// Delete the channel which is used to receive the MultipartReply message after all table features are received.
+		defer func() {
+			delete(b.mpartRplChs, mpartRequest.Xid)
+			close(tableFeatureChan)
```

Comment:
I don't think there is value in closing the channel here. If anything, it's a bit confusing to have a "receiver" close the channel.